### ISSUE TITLE
[inductor] Accuracy issue tracker

### ISSUE THREAD

**Issue by anijain2305** [pytorch/torchdynamo#1190]
Major deviations
- [ ] LayoutLMForMaskedLM - https://github.com/pytorch/torchdynamo/issues/1235
- [ ] crossvit_9_240 - https://github.com/pytorch/torchdynamo/issues/1233
- [ ] vit_base_patch16_224
- [ ] tinynet_a

Somewhat major deviations
- [ ] gluon_xception65
- [ ] spnasnet_100
- [ ] mobilenetv2_100

Repro - WIP - https://github.com/pytorch/torchdynamo/pull/1187

**Comment by desertfire** [comment]
I verified, these following 3 benchmarks are fixed on master. I do notice CUDAgraph disabled warning from Elias's recent checkin, I think that's likely why they got fixed.
```
crossvit_9_240
vit_base_patch16_224
tinynet_a
```

**Comment by desertfire** [comment]
I should clarify I used `--float32`, not sure if you are talking about `--amp`. It will be helpful to post the cmdline you used here.

**Comment by anijain2305** [comment]
I still see it at the `main` branch. My command is `python benchmarks/timm_models.py --accuracy --ci -d cuda --inductor --float32 --training --only=crossvit_9_240`

**Comment by anijain2305** [comment]
Closing in favor of #511
